dcs/mission.py

```diff
@@ -260,6 +260,9 @@ def loaddict(fname: str, mizfile: zipfile.ZipFile, reserved_files: List[str]) ->
 
         imp_mission = mission_dict["mission"]
 
+        # required modules
+        self.requiredModules = imp_mission.get("requiredModules", None)
```

Comment:
nit: `snake_case`

Reply:
n/p, thought to keep it in sync with what's in mission file.